.pre-commit-config.yaml

```diff
@@ -95,7 +95,8 @@ repos:
             polyfactory,
             prometheus_client,
             psycopg,
-            pydantic,
+            "git+https://github.com/pydantic/pydantic.git",
```

Comment:
Why do we do this `"git+https://github.com/pydantic/pydantic.git",` instead of declaring `pydantic`?

Reply:
To get the main branch directly. There were some typing issues there in the past. I can revert this to pydantic.